credential-status/src/test/java/me/uport/sdk/credentialstatus/Tests.kt

```diff
@@ -0,0 +1,12 @@
+package me.uport.sdk.credentialstatus
```

Comment:
the test packagename doesn't match the source packagename (`me.uport.sdk.credential_status`)